server/app/filters/OptionalProfileRoutes.java

```diff
@@ -10,7 +10,11 @@
 final class OptionalProfileRoutes {
 
   public static ImmutableList<String> routes =
-      ImmutableList.of("/", "/programs", "/applicants/programs");
+      ImmutableList.of(
+          "/",
+          "/programs",
+          "/applicants/programs",
+          controllers.routes.SupportController.handleUnsupportedBrowser().url());
```

Comment:
this lgtm and makes me thing we should make the other urls in this list reference the controller url instead of being hard-coded.